tidy3d/web/api/webapi.py

```diff
@@ -273,10 +274,15 @@ def start(
     task = SimulationTask.get(task_id)
     if not task:
         raise ValueError("Task not found.")
-    task.submit(
-        solver_version=solver_version,
-        worker_group=worker_group,
-    )
+
+    # note: this try block breaks if task has already been submitted, bypasses error.
+    try:
+        task.submit(
+            solver_version=solver_version,
+            worker_group=worker_group,
+        )
+    except WebErrorCore:
+        log.warning("Task has already been started, skipping.")
```

Comment:
I think it is not correct fix. 
the code need check the return code and then propose the correct message to final user.
Let's talk to @QimingFlex

Reply:
agree